src/kv/RocksDBStore.cc

```diff
@@ -2695,6 +2705,15 @@ class WholeMergeIteratorImpl : public KeyValueDB::WholeSpaceIteratorImpl {
     }
   }
 
+  std::string_view value_as_sv() override
+  {
+    if (smaller == on_main) {
+      return main->value_as_sv();
```

Comment:
a comment line reminding the reader what the equality means? (I know it is explained in the way 'smaller' is set, but still...)